stable/airflow/README.md

```diff
@@ -1,208 +1,209 @@
 
+For example, to add an AWS connection:
+```yaml
+airflow:
+  connections:
+  - id: my_aws
+    type: aws
+    extra: '{"aws_access_key_id": "**********", "aws_secret_access_key": "***", "region_name":"eu-central-1"}'
 ```
-http://mycompany.com/airflow/
-http://mycompany.com/airflow/flower
-```
-
-NOTE: Mounting the Airflow UI under a subpath requires an airflow version >= 2.0.x. For the moment
-(June 2018) this is **not** available on official package, you will have to use an image where
-airflow has been updated to its current HEAD. You can use the following image:
-`stibbons31/docker-airflow-dev:2.0dev`. It is rebase regularly on top of the `puckel/docker-airflow`
-image.
-
-Please also note that the Airflow UI and Flower do not behave the same:
 
-- Airflow Web UI behaves transparently, to configure it one just needs to specify the
-  `web.baseUrl` value. Ingress can be configured with `ingress.web.host` and `ingress.web.path`.
-- Flower cannot handle this scheme directly and requires a URL rewrite mechanism in front
-  of it. In short, it is able to generate the right URLs in the returned HTML file but cannot
-  respond to these URL. It is commonly found in software that wasn't intended to work under
-  something else than a root URL or localhost port. To use it, see the `values.yaml` for how
-  to configure your ingress controller to rewrite the URL (or "strip" the prefix path) and `flower.urlPrefix`.
+__NOTE:__ As connections may include sensitive data, we store the bash script which generates the connections in a Kubernetes Secret, and mount this to the pods. 
 
-  Note: unreleased Flower (as of June 2018) does not need the prefix strip feature anymore. It is
-  integrated in `docker-airflow-dev:2.0dev` image.
+__WARNING:__ Because some values are sensitive, you should take care to store your custom `values.yaml` securely before passing it to helm with: `helm -f <my-secret-values.yaml>`
 
-### Airflow configuration
+### Airflow-Configs/Variables
 
-`airflow.cfg` configuration can be changed by defining environment variables in the following form:
-`AIRFLOW__<section>__<key>`.
-
-See the
-[Airflow documentation for more information](http://airflow.readthedocs.io/en/latest/configuration.html?highlight=__CORE__#setting-configuration-options)
-
-This helm chart allows you to add these additional settings with the value key `airflow.config`.
-You can also add generic environment variables such as proxy or private pypi:
+We expose the `scheduler.variables` value to allow specifying [Airflow Variables](https://airflow.apache.org/docs/stable/concepts.html#variables) at deployment time, variables will be automatically imported by the Airflow scheduler when it starts up.
 
+For example, to specify an `environment` variable:
```

Comment:
can you add a remark the `variables` value is a string and not a structure.

maybe use the | syntax:
```yaml
airflow:
  variables: |
    environment: "dev"